asterius/rts/rts.unicode.mjs

```diff
@@ -0,0 +1,66 @@
+export class Unicode {
+    constructor(logger) {
+        Object.seal(this);
+    }
+
+    u_gencat(code) {
+        if (code >= 'A'.charCodeAt(0) && code <= 'Z'.charCodeAt(0)) {
+            return 0; // Uppercase letter
+        } else if (code >= 'a'.charCodeAt(0) && code <= 'z'.charCodeAt(0)) {
+            return 1; //Lowercase letter
+        } else if (code >= '0'.charCodeAt(0) && code <= '9'.charCodeAt(0)) {
+            return 8; //Decimal number
+        }
+    }
+
+    u_iswalpha(code) {
+        return (code >= 'A'.charCodeAt(0) && code <= 'Z'.charCodeAt(0)) ||
+            (code >= 'a'.charCodeAt(0) && code <= 'z'.charCodeAt(0));
+    }
+
+    u_iswalnum(code) {
+        return code >= '0'.charCodeAt(0) && code <= '9'.charCodeAt(0);
+    }
+
+    u_iswupper(code) {
```

Comment:
For upper/lower case conversion, we have `String.prototype.toUpperCase()`, please use that instead.